packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedFieldMessageEmbed.ts

```diff
@@ -0,0 +1,155 @@
+import { MessageEmbed, MessageEmbedOptions } from 'discord.js';
+
+import { PaginatedMessage } from './PaginatedMessage';
+
+/**
+ * This is a utility of {@link PaginatedMessage}, but paginating only the field of an embed.
+ * You must either use this class directly or extend it.
+ *
+ * @example
+ * ```typescript
+ * const { PaginationFieldMessageEmbed } = require('@sapphire/discord.js-utilities');
+ *
+ * new PaginationFieldManager()
+ *    .setTitleField('')
+ *    .setTemplate({ ...options })
+ *    .setItems([{ field: 'value' }, { field: 'value' }, { field: 'value' }, { field: 'value' }])
+ *    .setItemsPerPage(2)
+ *    .make()
+ *    .run(message.author, message.channel);
+ * ```
+ */
+export class PaginationFieldMessageEmbed<Element> extends PaginatedMessage {
+	private embedTemplate: MessageEmbed = new MessageEmbed();
+	private totalPages = 0;
+	private items: Element[] = [];
+	private itemsPerPage = 10;
+	private fieldTitle!: string;
```

Comment:
If title is optional and you're using a non-nullish operator here because you set `.setTitleField('')` in your example then remove the `!` and make this
```suggestion
	private fieldTitle = '';
```

Reply:
The title cannot be optional because it is necessary to create the field in the embeds.

Reply:
Then add a constructor with a title parameter that sets the property. Using non-null assertions should be avoided whenever possible.